deployment/media-insights-stack.yaml

```diff
@@ -122,11 +122,6 @@ Resources:
       ServiceToken: !GetAtt MieHelperFunction.Arn
       FunctionKey: "init_system_table"
 
-  # Custom log group
-  MieLogGroup:
-    Type: AWS::Logs::LogGroup
-    Properties:
-      RetentionInDays: 14
```

Comment:
i personally wouldn't drop it, i'd even go extra mile and turn it into an input parameter because the cost does spiral very quickly if you have a busy env.

Reply:
This resource isn't actually used anywhere. Thanks for the feedback, though.